locale/zh-cn/index.md

```diff
@@ -5,9 +5,9 @@ labels:
   current-version: 当前版本
   download: 下载
   download-for: 下载为
-  other-downloads: 其他下载
-  other-lts-downloads: 其他 LTS 下载
-  other-current-downloads: 其他最新版本下载
+  other-downloads: 其它下载
+  other-lts-downloads: 其它LTS版下载
```

Comment:
Make sure you have spaces between CJK and ASCII chararcters.

Reply:
OK, fixed.
PS: Why? The formation recognization?